src/vs/workbench/api/common/configurationExtensionPoint.ts

```diff
@@ -290,7 +290,7 @@ jsonRegistry.registerSchema('vscode://schemas/workspaceConfig', {
 			description: nls.localize('workspaceConfig.folders.description', "List of folders to be loaded in the workspace."),
 			items: {
 				type: 'object',
-				default: { path: '' },
+				defaultSnippets: [{ description: 'Path to folder', body: { path: '$1' } }],
```

Comment:
`Path to folder` would have to be localized (`nls.localize(..`)
Or just remove it, I don't think it adds much value.

Reply:
Sounds good to me. I'll just remove it.